c++/src/kj/CMakeLists.txt

```diff
@@ -270,12 +279,18 @@ if(BUILD_TESTING)
       compat/tls-test.c++
     )
     target_link_libraries(kj-heavy-tests kj-http kj-gzip kj-tls kj-async kj-test kj)
-    if (WITH_OPENSSL)
+    if(WITH_OPENSSL)
       set_property(
         SOURCE compat/tls-test.c++
         APPEND PROPERTY COMPILE_DEFINITIONS KJ_HAS_OPENSSL
       )
     endif()
+    if(WITH_FIBERS)
+      set_property(
+        SOURCE async-test.c++
+        APPEND PROPERTY COMPILE_DEFINITIONS KJ_USE_FIBERS
+      )
+    endif()
```

Comment:
This is the part failing the MSVC ci checks, but I am not sure why :man_shrugging:

Reply:
I don't understand why either, but since `KJ_USE_FIBERS` is now `PUBLIC`, you don't need to set the source file property on the test case, this if/set_property can be removed.

Reply:
Ok.